docs/containers.conf.5.md

```diff
@@ -434,8 +434,10 @@ and the logfile will not be rotated.
 
 **events_logger**="journald"
 
-Default method to use when logging events.
-Valid values: `file`, `journald`, and `none`.
+Default method to use when logging events.  For Red Hat Enterprise Linux
```

Comment:
I think the default for RHEL will be different between RHEL8 and RHEL9.  I think we should just say without a containers.conf it is journald, but you should check your containers.conf settings.